src/Payment/Service.php

```diff
@@ -3,18 +3,41 @@
 namespace PodPoint\Payments\Payment;
 
 use PodPoint\Payments\Payment;
+use PodPoint\Payments\Providers\Stripe\Payment\Exception;
+use PodPoint\Payments\Providers\Stripe\Customer\Service as CustomerService;
+use PodPoint\Payments\Providers\Stripe\Refund\Service as RefundService;
```

Comment:
the core payment service shouldn't be aware of any provider-specific things, there should be interfaces for each of these services